src/views/Receiver.vue

```diff
@@ -67,21 +67,28 @@ video
 #media-controls
   width: 60%
   animation: fade-in 0.4s
+  > *
```

Comment:
Instead of using * here can you use specific classes to make it more clear?

Reply:
okay! Fixed!